oop_basics/book.js

```diff
@@ -0,0 +1,49 @@
+export default class Book {
+    constructor(title, author, releaseYear) {
+        this._title = title;
+        this._author = author;
+        this._releaseYear = releaseYear;
+    }
+
+    get title() {
+        return this._title;
+    }
+
+    set title(newTitle) {
+        if (typeof newTitle !== 'string' || newTitle.trim() === '') {
```

Comment:
`if (typeof newTitle !== 'string' || !newTitle.trim().length) 
`
would be more convenient